netdata-installer.sh

```diff
@@ -1367,6 +1367,12 @@ detect_libc() {
 }
 
 should_install_ebpf() {
+  if [ -n "$EUID" ] && [ $EUID -ne 0 ]; then
```

Comment:
I have only one doubt, I could observe that the PR is doing what is expected, it is not doing download of the files, but we are not avoiding the compilation of the ebpf.plugin. Considering this, I think we need to change the message 

```
Not running as root (euid=0), not installing eBPF...
```

, because they will have `ebpf.plugin` inside `/usr/libexec/netdata/plugins.d`.

Reply:
Do we _want_ to avoid compliing the `ebpf.plugin` as well theN?

Reply:
To keep this exact message I think we should stop `ebpf.plugin` compilation, because the plugin will be compiled and it will always write an error when Netdata tries to load it. We could change the message for something like `Not running as root (euid=0), eBPF.plugin cannot be initialized ...`. If this happens, this PR can be merged.

Reply:
Yes - we want to avoid the compilation if it is not being installed.

Reply:
IMO, this should also translate to not building libbpf if we're not installing the eBPF plugin, so we'll also need a check there.

Reply:
@thiagoftsm Where are we at with the above? Do we skip compiling this at all if we don't have to? I can't tell 😳

Reply:
I think the best approach will be do not compile at all if it is not root.